src/main/java/com/rapid7/client/dcerpc/msvcctl/messages/RCreateServiceWResponse.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2017, Rapid7, Inc.
+ *
+ * License: BSD-3-clause
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *   Redistributions of source code must retain the above copyright notice,
+ * this list of conditions and the following disclaimer.
+ *
+ *  Redistributions in binary form must reproduce the above copyright
+ * notice, this list of conditions and the following disclaimer in the
+ * documentation and/or other materials provided with the distribution.
+ *
+ *  Neither the name of the copyright holder nor the names of its contributors
+ * may be used to endorse or promote products derived from this software
+ * without specific prior written permission.
+ *
+ *
+ */
+
+package com.rapid7.client.dcerpc.msvcctl.messages;
+
+import com.rapid7.client.dcerpc.io.PacketInput;
+import com.rapid7.client.dcerpc.messages.HandleResponse;
+
+import java.io.IOException;
+
+public class RCreateServiceWResponse extends HandleResponse{
+    private int tagId;
+
+    @Override
+    public void unmarshal(PacketInput packetIn) throws IOException {
```

Comment:
Use unmarshalResponse instead. Unmarshal will handle the ReturnCode

Reply:
Ok, I see the difference.  Fixed in my local branch.